terminusdb_client/woqlclient/connectionCapabilities.py

```diff
@@ -70,7 +70,7 @@ def author(self):
 
     def _databases_from_dbdocs(self):
         dbs = []
-        for docid in self.dbdocs.keys():
+        for docid in self.dbdocs():
```

Comment:
This is not correct... it should be self.dbdocs instead of self.dbdocs() as dbdocs is a dictionary, not a callable.